can/interface.py

```diff
@@ -27,21 +23,12 @@ def __new__(cls, other, channel=None, *args, **kwargs):
             can.rc['interface'] = kwargs['bustype']
```

Comment:
Yup I meant the library should treat it as read only - the user may of course change at will.